igor.dzetcyna/first/task 1/hash.rb

```diff
@@ -0,0 +1,14 @@
+# Знайти кількість ключів, значення яких починається на “yes”
+hash = {
+  'Yes' => 23, 
+  'b' => 'travel', 
+  'yesterday' => 34, 
+  5 => '234', 
+  yesss: :fg, 
+  try: 30, 
+  key: 'some value', 
+  ' yesterday1 '=> 34,
+  ' yesteryear '=> 2014}
+number = hash.select { |keys| keys.to_s.downcase.start_with? "yes" }
```

Comment:
it's better to name variables consistent to what is stored there, so I can look on a variable name and understand what's stored inside.

Reply:
**nitpick:** Also there is a way to refactor to use https://apidock.com/ruby/Enumerable/count so your code will be a little more compact

Reply:
Prefer single quoted strings.
https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals-single-quote